examples/options/src/details/configuration_category.cpp

```diff
@@ -0,0 +1,63 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023-2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Local include(s).
+#include "configuration_category.hpp"
+
+// System include(s).
+#include <algorithm>
+#include <format>
+#include <iostream>
+
+namespace traccc::opts {
+
+configuration_category::configuration_category(std::string_view name)
```

Comment:
We are not seriously splitting up an 80-line header file and a 100-line C++ file into 3 translation units and 4 header files, right? To be frank that's just silly...